tests/court_tests/test_court.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from court.models import Court
+
+
+TEST_ID = 1
+TEST_X = 1.234567
+TEST_Y = 9.876543
+TEST_CITY = 'New York'
+TEST_NEIGHBORHOOD = 'Hogwarts'
+TEST_MAX_PLAYERS = 5
```

Comment:
I have some reservations about creating constants for test files on the global scope.

The reason being that those are not actually constants. They could be changed by one of the test cases and then affect all other test cases.

One approach is to define them as fixtures so each test case gets a fresh copy, but I'm not sure you actually need them defined in the global scope at this point.

Reply:
Deleted. Thank you for your comment.